regen-network/regen-web
# feat: APP-111 list project CTA

## Description

https://regennetwork.atlassian.net/browse/APP-111

Needs https://github.com/regen-network/regen-server/pull/464 for list project + google login to work

Targeting `main` so we get have it deployed right away.



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2347--regen-marketplace.netlify.app/
1. Click "list a project" and log in via Keplr with an address that is not a class issuer, it should redirect you to the basic info form
2. Click "list a project" and log in via Keplr with an address that is a class issuer, it should redirect you to the choose a credit class form
3. Click "list a project" and log in via email, it should redirect you to the basic info form
4. **Please do not test this step before https://github.com/regen-network/regen-server/pull/464 is merged:** Click "list a project" and log in via google, it should redirect you to the basic info form
5. Logging in with the regular "log in" button should be unchanged

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
